### ISSUE TITLE
Some problem happens when we store one image using vutils.save_image

### ISSUE THREAD

**Issue by MeowTsukiko** [AlexiaJM/Deep-learning-with-cats#23]
When we generate the same number of images as batch_size during training, it performs very well. But when we only want to generate one picture, there is a problem with its brightness. We also set the batch_size to 64 when generating the picture, but this problem still exists when saving a picture. We think it is the normalization of the following line of code.
`vutils.save_image(fake_test.data[0:1,:,:,:], './output/%01d.png' % i , normalize=True)`
How can we store one image at a time?
We are looking forward to your reply.

**Comment by AlexiaJM** [comment]
This annoying behavior is due to the batch norm. It is best that you generate a full mini-batch and then extract a single image.  Otherwise, you could try replacing the batch norm with instance norm, or try without batch norm.
